src/main/java/mods/eln/sixnode/thermalcable/ThermalCableDescriptor.java

```diff
@@ -109,8 +109,8 @@ public void addInformation(ItemStack itemStack, EntityPlayer entityPlayer, List
         list.add(tr("Serial resistance: %1$K/W", Utils.plotValue(thermalRs * 2)));
         list.add(tr("Parallel resistance: %1$K/W", Utils.plotValue(thermalRp)));
         list.add("");
-        Collections.addAll(list, tr("Low serial resistance\n => High conductivity.").split("\n"));
-        Collections.addAll(list, tr("High parallel resistance\n => Low power dissipation.").split("\n"));
+        Collections.addAll(list, tr("Low serial resistance\n ⇨ High conductivity.").split("\n"));
+        Collections.addAll(list, tr("High parallel resistance\n ⇨ Low power dissipation.").split("\n"));
```

Comment:
Non-blocking: Maybe `->`? I'm not sure how good Minecraft is at rendering unicode and it would probably look out of place.

Reply:
In order to maintain visual consistency before and after the modification, I replaced "=>" with "⇒". Perhaps due to my usage of the SmoothFont mod, I haven't noticed any Unicode rendering errors. Maybe using "->" would be better.I will modify "⇒" to "->" later.